README.md

```diff
@@ -107,7 +107,8 @@
 - [InKino](https://github.com/roughike/inKino) - A multiplatform Dart movie app by [Iiro Krankka](https://github.com/roughike).
 - [Music Player](https://github.com/iampawan/Flutter-Music-Player) - Full featured music player by [Pawan Kumar](https://about.me/imthepk).
 - [Cinematic](https://github.com/aaronoe/FlutterCinematic) - UI for Movie DB Public API by [Aaron Oertel](https://github.com/aaronoe).
-- [TuneIN](https://github.com/moda20/flutter-tunein) - Dynamic themed music player with casting [Moda20](https://github.com/moda20).
+- [TuneIN](https://github.com/moda20/flutter-tunein) - Dynamic themed music player with casting [Moda20](https://github.com/moda20).\
```

Comment:
Please remove this Backslash @SaadAAkash

Reply:
Removed in the latest [commit](https://github.com/tortuvshin/open-source-flutter-apps/pull/44/commits/82ec59da113ac2c1e212c336e849311f35991d94) to this PR. @tortuvshin thanks for noticing.